torchmetrics/text/bert.py

```diff
@@ -182,16 +186,20 @@ def __init__(
                     "`BERTScore` metric with default tokenizers requires `transformers` package be installed."
                     " Either install with `pip install transformers>=4.0` or `pip install torchmetrics[text]`."
                 )
-            if not model_name_or_path:
-                model_name_or_path = _DEFAULT_MODEL
+            if model_name_or_path is None:
                 warn(
-                    "The argument `model_name_or_path` was not specified while it is required when default "
+                    "The argument `model_name_or_path` was not specified while it is required when default"
                     " `transformers` model are used."
                     f"It is, therefore, used the default recommended model - {_DEFAULT_MODEL}."
```

Comment:
```suggestion
                    "The argument `model_name_or_path` was not specified while it is required when the default"
                    " `transformers` model is used."
                    f" It will use the default recommended model - {_DEFAULT_MODEL!r}."
```